athena/modules/modeling/module_modeling_llm/module_modeling_llm/apollon_transformer/apollon_json_transformer.py

```diff
@@ -44,6 +46,9 @@ def transform_json(model: str) -> tuple[str, dict[str, str], str, dict[str, str]
         # Convert legacy version 2.0.0 to version 3.0.0 if needed
         if model_dict.get("version") == "2.0.0":
             model_dict = ApollonJSONTransformer._convert_v2_to_v3(model_dict)
```

Comment:
Can you please also move the conversion from v2 to v3 into the schema converter class?
Then all conversions are in the same class and it is easier to find and understand :)